core/modules/parsers/wikiparser/wikiparser.js

```diff
@@ -208,10 +223,13 @@ WikiParser.prototype.parseBlock = function(terminatorRegExpString) {
 	// Look for a block rule that applies at the current position
 	var nextMatch = this.findNextMatch(this.blockRules,this.pos);
 	if(nextMatch && nextMatch.matchIndex === this.pos) {
-		return nextMatch.rule.parse();
+		return this.parseRule(nextMatch.rule);
 	}
 	// Treat it as a paragraph if we didn't find a block rule
-	return [{type: "element", tag: "p", children: this.parseInlineRun(terminatorRegExp)}];
+	var start = this.pos;
+	var children = this.parseInlineRun(terminatorRegExp);
+	var end = this.pos;
+	return [{type: "element", tag: "p", children, start, end }];
```

Comment:
Apologies for the delay -- the TW5 core sticks to ECMAScript 5 for compatibility with older browsers, and so we can't use destructuring like this. Instead we should use:

```
return [{type: "element", tag: "p", children: children, start: start, end: end }];
```

Reply:
@SmilyOrg ... can you add the requested changes and push a new version. So the tests can be run again. So I can see, where the test error came from.

Reply:
@pmario IIRC most if not all the test errors are from the fact that the tests assume that the parser will not return start and end ranges but now it does. So the fixtures that we compare against need updating.